test/e2e/aws_test.go

```diff
@@ -368,8 +370,57 @@ var _ = Describe("functional tests", func() {
 			deleteCluster(setup.namespace, setup.clusterName)
 		})
 	})
+
+	Describe("Cluster should be highly available", func() {
+		It("", func() {
+			By("Creating a workload cluster with single control plane")
+			setup.multipleAZ = true
+			clusterK8sClient := makeSingleControlPlaneCluster(setup)
+
+			By("Creating the second Control Plane Machine in second AZ")
+			awsMachineName := setup.cpAWSMachinePrefix + "-1"
+			bootstrapConfigName := setup.cpBootstrapConfigPrefix + "-1"
+			machineName := setup.cpMachinePrefix + "-1"
+			setup.availabilityZone = availabilityZones[1]
+			createAdditionalControlPlaneMachine(setup, machineName, awsMachineName, bootstrapConfigName)
+
+			By("Creating the MachineDeployment in second AZ")
+			setup.availabilityZone = availabilityZones[1]
+			setup.subnetId = nil
+			createMachineDeployment(setup)
+
+			By("Creating the LB service")
```

Comment:
This is a little confusing, we should explain that this is a kubernetes service and not an AWS ELB